x/incentives/types/gov.go

```diff
@@ -0,0 +1,73 @@
+package types
+
+import (
+	"fmt"
+	"strings"
+
+	govtypes "github.com/cosmos/cosmos-sdk/x/gov/types"
+)
+
+const (
+	ProposalTypeCreateGaugeGroups = "CreateGaugeGroups"
+)
+
+func init() {
+	govtypes.RegisterProposalType(ProposalTypeCreateGaugeGroups)
+	govtypes.RegisterProposalTypeCodec(&CreateGaugeGroupsProposal{}, "osmosis/CreateGaugeGroupsProposal")
+}
+
+var (
+	_ govtypes.Content = &CreateGaugeGroupsProposal{}
+)
+
+// NewCreateGaugeGroupsProposal returns a new instance of a gauge group creation proposal struct.
+func NewCreateGaugeGroupsProposal(title, description string, groups []CreateGroup) govtypes.Content {
```

Comment:
Ditto for rename in this file

Reply:
https://github.com/osmosis-labs/osmosis/pull/6547/commits/cb2b2a0756528d492c159ed468ef42d763c51cb3